gravitee-am-ui/src/app/domain/settings/users/user/profile/profile.component.html

```diff
@@ -21,46 +21,69 @@
       <form (ngSubmit)="update()" fxLayout="column" #userForm="ngForm">
         <div class="gv-form-section">
           <div class="gv-form-section-title">
-            <h5>Username</h5>
-            <small>The username of the user.</small>
+            <h3>{{ isServiceAccount() ? 'Service Name' : 'Username' }}</h3>
+            <small>{{ isServiceAccount() ? 'The name of service.' : 'The username of the user.' }}</small>
             <mat-divider></mat-divider>
           </div>
           <div fxLayout="column">
             <div fxLayout="column">
               <div fxLayout="column">
                 <form (ngSubmit)="updateUsername()" #usernameForm="ngForm" fxLayout="row">
                   <mat-form-field fxFlex="70" appearance="outline" floatLabel="always">
-                    <mat-label>Username</mat-label>
+                    <mat-label>{{ isServiceAccount() ? 'Service Name' : 'Username' }}</mat-label>
                     <input
                       matInput
                       type="text"
                       required
                       #username="ngModel"
                       name="username"
                       [(ngModel)]="user.username"
-                      placeholder="Updated Username"
+                      placeholder="{{ isServiceAccount() ? 'Update Service Name' : 'Update Username' }}"
```

Comment:
it can be extracted to method

Reply:
I dont think it make sens to create metod only for this use.

Reply:
it can be reused in ts file